joystick_interface_auv:
  ros__parameters:
    surge_scale_factor: 60.0
    sway_scale_factor: 60.0
    yaw_scale_factor: 25.0
    heave_scale_factor: 17.5
    roll_scale_factor: 30.0
    pitch_scale_factor: 20.0

    surge_scale_ref_param: 5000
    sway_scale_ref_param: 5000
    heave_scale_ref_param: 5000
    roll_scale_ref_param: 5000
    pitch_scale_ref_param: 5000
    yaw_scale_ref_param: 5000
